packages/libs/app-sdk/package.json

```diff
@@ -68,7 +68,7 @@
     "*.md"
   ],
   "devDependencies": {
-    "@lit-protocol/pkp-ethers": "^7.2.0",
+    "@lit-protocol/pkp-ethers": "^7.2.3",
     "@lit-protocol/types": "^7.0.8",
```

Comment:
Can we update types in here too?

Reply:
sure, i updated the PR to do it everywhere including here